Updated image in flex-basis doc

asummers1 opened the pull request:
<!-- Please provide the following information to help us review this PR: -->

> What was wrong/why is this fix needed? (quick summary only)
This new image is sharper and has a higher bit depth than the previous (32 vs 24), which makes it more readable
> MDN URL of main page changed
https://developer.mozilla.org/en-US/docs/Web/CSS/flex-basis
> Issue number (if there is an associated issue)

> Anything else that could help us review it

asummers1 committed:
Delete firefox-flex-basis.jpg

asummers1 committed:
Add files via upload

asummers1 committed:
Updated flex-basis

eatmorepies-dev commented:
Changed the index page to represent the new file (a .png file extension compared to .jpg)

rachelandrew commented:
Hi @eatmorepies-dev it looks like this is failing because the image needs more compression.  You can do that with yarn 

`yarn filecheck "files/en-us/web/css/flex-basis/firefox-flex-basis.png" --save-compression`

Or use something like Squoosh https://squoosh.app/

sideshowbarker committed:
Compress file

Ran yarn filecheck "files/en-us/web/css/flex-basis/firefox-flex-basis.png" --save-compression

sideshowbarker commented:
Alexander, thanks much, and congrats on landing your first docs change here — welcome aboard 🎉
